change/@microsoft-teams-js-428c45d4-9a6f-46ff-8c5a-33db61803db2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Updates to the URLs for docs links.",
```

Comment:
Can you please update the comment to use  past tense, e.g., "Updated the URLs for docs links." Thanks!

Reply:
I just quickly jumped into the github editor and did this